delivery_price_rule_untaxed/models/delivery_carrier.py

```diff
@@ -11,14 +10,20 @@ class DeliveryCarrier(models.Model):
     _inherit = 'delivery.carrier'
 
     @api.multi
-    def get_price_available(self, order):
+    def _get_price_available(self, order):
         self.ensure_one()
+
+        delivery_line = order.order_line.filtered(
+            lambda line: line.is_delivery
+        )
+        delivery_price = delivery_line and delivery_line.price_subtotal or 0.0
```

Comment:
What if there is more than one?

Reply:
This is never the case as only one is created by module `delivery`

Reply:
Then please add a `delivery_line[0]` or another kind of check/filter somewhere, otherwise this line would raise an exception if `delivery_line` has more than one record

Reply:
Sure